bag_of_words/code/bag_of_words.py

```diff
@@ -0,0 +1,36 @@
+# Read papers and convert them to bag-of-words representations.
+
+indir = "../data/raw/pubmed-txts-mf/"

```

Comment:
I recommend following [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines for Python code style. 

Here are some examples:
- please use CAPITAL_LETTERS for constant var names.
- please put `import` statements first in the code
- please do not initialize objects outside of a function (or method, if you use classes)

For code that can be called as a module or stand-alone, please use the `if __name__ == "__main__"` convention.

Reply:
I have enhanced the coding style of all scripts except classifiers.py which is under active experimentation.